cms/static/sass/elements/_modules.scss

```diff
@@ -180,6 +180,37 @@
         display: inline-block;
       }
     }
+
+    .support-documentation {
```

Comment:
I have some additional styling tweaks to do, and I have to check if there are any responsive issues.